src/core/thread/mle_router.cpp

```diff
@@ -3802,6 +3802,11 @@ otError MleRouter::CheckReachability(uint16_t aMeshSource, uint16_t aMeshDest, I
         {
             ExitNow();
         }
+        else if (IsAnycastLocator(aIp6Header.GetDestination()))
```

Comment:
:+1: I think this is a great idea.

Reply:
Should we also limit the frequency of sending the server data notification here?

Reply:
Good suggestion. Updated in new patch